hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerRatisRequest.java

```diff
@@ -29,30 +32,38 @@
 import org.apache.hadoop.ozone.om.ratis.utils.OzoneManagerRatisUtils;
 import org.apache.hadoop.ozone.om.request.OMRequestTestUtils;
 import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos;
+import org.apache.hadoop.ozone.protocolPB.OzoneManagerProtocolServerSideTranslatorPB;
 import org.junit.Assert;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
 import org.junit.rules.TemporaryFolder;
 import org.mockito.Mockito;
 
+import java.io.IOException;
 import java.util.ArrayList;
 
+import static org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos.Status.INVALID_REQUEST;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.when;
 
 /**
- * Test: Creating a client request for a bucket which doesn't exist.
+ * Test OM Ratis request handling.
  */
 public class TestOzoneManagerRatisRequest {
   @Rule public TemporaryFolder folder = new TemporaryFolder();
 
   private OzoneManager ozoneManager;
   private OzoneConfiguration ozoneConfiguration = new OzoneConfiguration();
   private OMMetadataManager omMetadataManager;
+  private OzoneManagerRatisServer ratisServer;
+  private ProtocolMessageMetrics<ProtocolMessageEnum> protocolMessageMetrics;
+  private long lastTransactionIndexForNonRatis;
 
```

Comment:
Why do these get declared here?  They seem to be local variables. Is this for future use?

Reply:
Thank you for pointing this out, we may not use them in the future, made changes to declare them locally.